script.js

```diff
@@ -38,6 +38,33 @@ const showSubmitLoader = (show = true) => {
   }
 };
 
+const startedDate = document.getElementById("startedOn");
+const endDate = document.getElementById("endsOn");
+
+const setEndDate = startDate => {
+  const startTimeEpoch = new Date(startDate).getTime();
+  const endTime = new Date(startTimeEpoch + 1000 * 60 * 60 * 24 * 7);
+  const dd = String(endTime.getDate()).padStart(2, "0");
+  const mm = String(endTime.getMonth() + 1).padStart(2, "0");
+  const yyyy = endTime.getFullYear();
+
+  document.getElementById("endsOn").value = `${yyyy}-${mm}-${dd}`;
+}
+
+document.getElementById("endsOn").addEventListener("change", (event) => {
+  if (event.target.value) {
+    if (startedDate.value < endDate.value) {
+      alert("End Date should be greater than the Start Date")
+    }
+  }
+})
+
+document.getElementById("startedOn").addEventListener("change", function (event) {
```

Comment:
We've declared startedDate and endDate but we're not using it on line 51, 54, 62... 🤔